docs/src/components/pricing/HeroPricing.tsx

```diff
@@ -10,7 +10,7 @@ export default function HeroPricing() {
     <Container>
       <Box
         sx={{
-          height: '40vh',
+          height: '28vh',
```

Comment:
To move with smaller steps

```suggestion
          height: '32vh',
```